apps/api/src/types/index.ts

```diff
@@ -1,492 +1,4 @@
 import { Request } from "express";
```

Comment:
is this line the only line left in this file?  Should this file just be deleted?